esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4552,8 +4552,14 @@ int CWsWorkunitsSoapBindingEx::onStartUpload(IEspContext &ctx, CHttpRequest* req
                 throw MakeStringException(ECLWATCH_INVALID_INPUT, "Only one WU ZAP report is allowed.");
 
             VStringBuffer fileName("%s%s", zipFolder, fileNames.item(0));
+#ifdef _CONTAINERIZED
+            VStringBuffer importDir("%s%simport", wswService->getQueryDirectory(), PATHSEPSTR);
```

Comment:
future: all dir. creation of this sort (inside importWorkUnit) should ensure it creates a truly unique dir, by using something like mkdtemp. In theory I could import >1 workunit with same name (but were different, e.g. from diff. envs) that would clash at the moment.

Reply:
OK. 
In this PR, a subfolder under the import folder will be created. The subfolder is named using the WUID. The workunit is created when the ZAP report is imported and the WUID is unique.

Reply:
ah ok, if wuid is new (and hence guaranteed to be unique), this is a non-issue